docs/topics/db/managers.txt

```diff
@@ -214,11 +214,13 @@ appropriate for your circumstances, you can tell Django which class to use by
 setting :attr:`Meta.base_manager_name
 <django.db.models.Options.base_manager_name>`.
 
-Base managers aren't used when querying on related models. For example, if the
-``Question`` model :ref:`from the tutorial <creating-models>` had a ``deleted``
-field and a base manager that filters out instances with ``deleted=True``, a
-queryset like ``Choice.objects.filter(question__name__startswith='What')``
-would include choices related to deleted questions.
+Base managers aren't used when querying on related models, or when
+:ref:`accessing a to-many relationship <backwards-related-objects>`. For
```

Comment:
`backwards-related-objects` is only about one-to-many relationship, so I think it would be good to mention both here:
```
... or when :ref:`accessing a one-to-many or many-to-many relationship <backwards-related-objects>`. ....
```

Reply:
I can update this patch if you agree :wink:

Reply:
Yeah, fine. I messed around a few times with different wording. 

There's no exact bit that covers both, but `to-many` work the same whether it's 1toM or M2M, so I thought the discussion there was probably what's needed. 

Please do update. Thank you sir. 🙂

Reply:
Thanks :+1: